src/test/java/emissary/core/BaseDataObjectTest.java

```diff
@@ -1463,4 +1463,39 @@ void testNewInputStream() throws IOException {
             assertArrayEquals(bytes2, byteArrayOutputStream.toByteArray());
         }
     }
+
+    @Test
+    void testGetParameterAsString() {
+        this.b.putParameter("A", 1L);
+        this.b.appendParameter("A", "TWO");
+        this.b.appendParameter("A", "THREE");
+        assertEquals("1", this.b.getParameterAsString("A"));
```

Comment:
Is it worth checking for the log message?
```suggestion
        try (LogbackTester logbackTester = new LogbackTester(BaseDataObject.clas
s.getName())){
            assertEquals("1", this.b.getParameterAsString("A"));
            LogbackTester.SimplifiedLogEvent logEvent = new LogbackTester.Simplifi
edLogEvent(Level.WARN, "Multiple values for parameter, returning the first - par
ameter:A, number of values:3",null);
            logbackTester.checkLogList(Collections.singletonList(logEvent));
        }
```